src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -243,3 +243,24 @@ def load_command_table(self, _):
         g.custom_command('create', 'aks_trustedaccess_role_binding_create')
         g.custom_command('update', 'aks_trustedaccess_role_binding_update')
         g.custom_command('delete', 'aks_trustedaccess_role_binding_delete', confirmation=True)
+
+    # AKS mesh commands
+    with self.command_group('aks mesh', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
+        g.custom_command(
+            'enable',
+            'aks_mesh_enable',
+            supports_no_wait=True)
```

Comment:
Better test the `--no-wait` option in test. Sometimes you will find that the parameter name is not the same as expected.

Reply:
I have manually tested all these commands with `--no-wait`.  Worked well.